### DISCUSSION TITLE
EV total Kwh charged - strange plott in HA

### DISCUSSION THREAD

**Discussion by Xnygra** [serkri/SmartEVSE-3#238] [Q&A] [answered]
When plotting total charged kvh in Home assistant i get zero values that mess up the graph, not sure where the problem lies, in SmartEvse,  HA or me..... I guess it is the latter. Anyway, 

I have an Estron EV meter connected and i get the data to HA through HA-smart EVSE integration AND through MQTT, and both have the same strange behavior. 
Anyone of you smart guys have any idea how to remove those zeros? 

![image](https://github.com/serkri/SmartEVSE-3/assets/44607420/a3b46c57-680a-416a-9775-ff1163e1b331)

**Comment by dingo35** [comment, marked as answer]
Try one of the new 3.5 alpa releases, it has code removed that is suspected to mess up API values....

**Reply by Xnygra** [reply]
Thank you, I have updated to 3.5 now, so I will see over the next few days if it have solved the issue.

**Reply by Xnygra** [reply]
Seems to work, have tried with a few charging sessions and different vehicles, but i'll give it a week to see if there is any glitches, if not, it would be a good candidate for update the main branch.

**Reply by dingo35** [reply]
Ehmm, new 3.5 IS the new main branch.....

**Reply by HansNetcom** [reply]
Hi. I,m also updated to 3.5.
Works fine 👍
